auslib/web/admin/swagger/api.yaml

```diff
@@ -1893,82 +1916,956 @@ paths:
+        description: Returns scheduled changes for required signoffs of products.
+      parameters:
+        - $ref: '#/parameters/allSCParam'
+      responses:
+        '200':
+          description: Returns JSON object of list of all scheduled changes for required signoffs of products plus total count
+          schema:
+            type: object
+            required:
+              - scheduled_changes
+            allOf:
+              - $ref: "#/definitions/CountModel"
+              - type: object
+                properties:
+                  scheduled_changes:
+                    description: array where each element is a scheduled change for a Product Required Signoff.
+                    type: array
+                    minItems: 0
+                    items:
+                      type: object
+                      allOf:
+                        - $ref: '#/definitions/SCChangeType'
+                        - $ref: '#/definitions/SCTime'
+                        - $ref: '#/definitions/SCIdModel'
+                        - $ref: '#/definitions/SCDataVersion'
+                        - $ref: '#/definitions/SCGetModel'
+                        - $ref: '#/definitions/DataVersionNullable'
+                        - $ref: '#/definitions/SignoffsRequiredNullable'
+                        - $ref: '#/definitions/ProductModel'
+                        - $ref: '#/definitions/RoleModel'
+                        - $ref: '#/definitions/ChannelModel'
+                      required:
+                        - change_type
+                        - data_version
+                        - when
+                        - product
+                        - channel
+                        - role
+                        - signoffs_required
+    post:
+      summary: create scheduled change for required signoffs of Product.
+      description: >
+        Create scheduled change for required signoffs of product(existing or new). [Docs](http://mozilla-balrog.readthedocs.io/en/latest/admin_api.html#id34)
+      tags:
+        - Scheduled Changes
+      operationId: auslib.web.admin.views.mapper.scheduled_changes_rs_product_post
+      consumes:
+        - application/json
+      produces:
+        - text/html
+        - application/json
+      externalDocs:
+        url: "http://mozilla-balrog.readthedocs.io/en/latest/admin_api.html#scheduled-changes-object"
+        description: "create scheduled change for required signoffs of product."
+      parameters:
+        - name: sc_rs_product_body
+          in: body
+          description: request body for sc required signoffs of product
+          required: true
+          schema:
+            type: object
+            allOf:
+              - $ref: '#/definitions/CSRFModel'
+              - $ref: '#/definitions/SCChangeType'
+              - $ref: '#/definitions/SCTime'
+              - $ref: '#/definitions/DataVersionModel'
```

Comment:
I think using DataVersionModel here is causing some problems with the UI. When I tried to add a new role requirement for an existing signoff I got this: https://screenshots.firefox.com/XPk0lWEWCYG1HH15/localhost

The requests we make for changes like that contain the following POST data:

> {"product":"Firefox","role":"relman","csrf_token":"xxxxxxx","data_version":null,"channel":"beta","when":1501599718293,"change_type":"insert","signoffs_required":"1"}

Note that data_version is null because adding a new required signoff ends up a new row in the database - not modifying an existing one.